modular_skyrat/modules/ghostcafe/code/ghost_role_spawners.dm

```diff
@@ -28,8 +28,8 @@
 		var/area/A = get_area(src)
 		//new_spawn.AddElement(/datum/element/ghost_role_eligibility, free_ghosting = TRUE) SKYRAT PORT -- Needs to be completely rewritten
 		new_spawn.AddElement(/datum/element/dusts_on_catatonia)
-		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type, /area/misc/hilbertshotel, /area/centcom/holding/cafe, /area/centcom/holding/cafewar, /area/centcom/holding/cafebotany,
-		/area/centcom/holding/cafebuild, /area/centcom/holding/cafevox, /area/centcom/holding/cafedorms, /area/centcom/holding/cafepark, /area/centcom/holding/cafeplumbing))
+		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type, /area/misc/hilbertshotel, /area/centcom/holding/cafe,
```

Comment:
Hilberts Hotel is a valid location reachable by normal players through a space ruin, I do not recommend including it here

Reply:
That was already there, I didn't mess with that.
The ghost cafe already has access to hilbert's hotel, and a ton of people use the hotel rooms there for scenes, so we wouldn't be removing that.